### ISSUE TITLE
Add Sparkle updater in the app

### ISSUE THREAD

**Issue by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#935]
This will require an update feed to be generated. We can either do it statically and manually update it when we do a release, or build it into the server.

It's probably easiest to do it manually for now, but I'd rather not generate the AppCast XML by hand, so maybe we represent releases in a YML (or similar) file in the app and have the server generate an AppCast from that.

How do you do it for Hummingbird @finestructure?

**Comment by finestructure** [comment]
I've tried adding Sparkle to HB, again as recently as last week but the app crashes on launch after embedding it. I'm not sure if it's an M1 thing or if I'm just doing it wrong. I've had Sparkle integration in my CronniX app 15 years ago but I've now failed to do this on at least three occasions in Hummingbird 😅

I believe there's a tool that comes with Sparkle that create the AppCast XML from some input, so that shouldn't be a problem.
